Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfigurationInternal.h

```diff
@@ -50,3 +53,13 @@ _WKDragLiftDelay toDragLiftDelay(NSUInteger);
 _WKDragLiftDelay toWKDragLiftDelay(WebKit::DragLiftDelay);
 WebKit::DragLiftDelay fromWKDragLiftDelay(_WKDragLiftDelay);
 #endif
+
+#endif // __cplusplus
+
+@interface WKWebViewConfiguration (NonCpp)
+
++ (/* nullable */ NSString *)_canonicalSchemeForURLScheme:(NSString *)urlScheme;
```

Comment:
Why not actually mark this `nullable`?

Reply:
I tried that originally, but I got a lottt of

```
Pointer is missing a nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified)
```

elsewhere in the file

Reply:
I think you just need `NS_ASSUME_NONNULL_BEGIN` / `NS_ASSUME_NONNULL_END`?